server/routes/api/team.ts

```diff
@@ -60,4 +62,76 @@ router.post("team.update", auth(), async (ctx) => {
   };
 });
 
+router.post("teams.create", auth(), async (ctx) => {
```

Comment:
I need a follow up issue to pluralize the other team endpoints and deprecate the singular.